lib/worker/src/db/publications.rs

```diff
@@ -61,6 +81,24 @@ pub async fn pub_table_insert(pool: &PgPool, stmts: Vec<String>) -> Result<()> {
     Ok(txn.commit().await?)
 }
 
+/// Returns cache config
+pub async fn get_cache_config(pool: &PgPool, ns: &str, rel: &str) -> Result<Option<i64>> {
```

Comment:
what will this do for existing pubs that don't have a cache config?

Reply:
good question. it will probably fail, we can create empty entries for existing pubs or change the query to check for that

Reply:
probably easiest to change the query result / logic to know about some pubs not having a policy